docs/configuration.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[configuration]]
+== Configuration
+
+Configure the agent with `AgentConfiguration` in the agent's `start()` function.
+
+// some config example that preferably is correct unlike mine
+[source,swift]
+----
+let config =  AgentConfiguration()
+Agent.start(with:config)
```

Comment:
Is it clear enough to users how to use the configuration options below? I don't know Swift, so it's unclear to me how the config options listed below fit into this example.

Reply:
I added more clear examples of usage. What do you think?

Reply:
Nice